dunamai/__init__.py

```diff
@@ -337,9 +375,6 @@ def __init__(
         #: Optional PEP 440 epoch.
         self.epoch = epoch
 
-        self._matched_tag = None  # type: Optional[str]
-        self._newer_unmatched_tags = None  # type: Optional[Sequence[str]]
```

Comment:
These are used by Dunamai's CLI. It's a bit hacky, but we do need them.

Reply:
They are still assigned, just higher up in the function